modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-2053[OSSM-2053] In Service Mesh 2.1.5 and earlier and 2.2.2 and earlier, during SMCP reconciliation, the SMMR controller removed the member namespaces from `SMMR.status.configuredMembers`. This caused the services in the member namespaces to become unavailable for a few moments.
++
+In Service Mesh 2.1.5.1 and later and 2.2.3 and later, the SMMR controller no longer removes the namespaces from `SMMR.status.configuredMembers`. Instead, the controller adds the namespaces to `SMMR.status.pendingMembers` to indicate that they are not up-to-date. During reconciliation, as each namespace synchronizes with the SMCP, the namespace is automatically removed from `SMMR.status.pendingMembers`.
```

Comment:
"During reconciliation, as each namespace synchronizes with the SMCP, the namespace is automatically removed from `SMMR.status.pendingMembers`"

And moved to `SMMR.status.configuredMembers`???  What happens when the namespace is removed from `SMMR.status.pendingMembers`?

Reply:
@JStickler from what Marko explained to me that's essentially when something becomes available.  I don't think there is much visible to the user.

@luksa when you are online can you provide more details? We want to release on 10/4/22.

Reply:
@rh-tokeefe , @luksa I'm guessing that the namespace is moved to `SMMR.status.configuredMembers`, but I'd want Marko to confirm that.

Reply:
Tim and I chatted and I got my question answered.